src/wwwroot/api/scripts/GetExperimentsOfPhysics.js

```diff
@@ -6,25 +9,67 @@ $(document).ready(function () {
     $.get('/Classrooms/GetPhysicsSubmissionOfTheStudent', { studentId: studentId, classroomId : classroomId }, function(data) {
         for (let i=0; i< data.length; i++) {
             InitiateExperiments(data[i]);
+            submissionTableApi.push(data[i]);
         }
+        //console.log($('#exp03Phy').html());
     });
+    
+    
+    
+    $('.btnPhy').click(function () {
+        let btnIdValue = this.id;
+        let clickedTableID = btnIdValue.replace("btn", "exp");
+        let clickedTableCurrentLength = initialLengthOfPhyExp[clickedTableID][clickedTableID];
+        
+        //let submissionAPIIndex = Object.keys(submissionTableApi["experiment"]).indexOf(keytoFind);
+        let tBody = findBaseTBody(clickedTableID);
+
+        clickedTableCurrentLength++;
+        
+        tBody = tBody.replace(/"group"/g, "'group"+ clickedTableCurrentLength + "'");
+        tBody = tBody.replace(/"typeOf"/g, "'typeOf"+ clickedTableCurrentLength + "'");
+        
+        initialLengthOfPhyExp[clickedTableID][clickedTableID] = clickedTableCurrentLength;
+        
+        //console.log(tBody);
+        $('#'+clickedTableID).append('<tr>' + tBody + '</tr>');
```

Comment:
This line has a minor bug. It's adding unnecessary extra **table row** when there is any rowspan in tBody. **I'll fix this.**

Reply:
oh, shit !!!